tests/rules/prefer-concat-latest-from.test.ts

```diff
@@ -1,150 +1,290 @@
 import { stripIndent } from 'common-tags'
+import { fromFixture } from 'eslint-etc'
 import path from 'path'
 import rule, {
   messageId,
-  messageIdSuggest,
 } from '../../src/rules/effects/prefer-concat-latest-from'
 import { NGRX_MODULE_PATHS } from '../../src/utils'
 import { ruleTester } from '../utils'
 
-ruleTester().run(path.parse(__filename).name, rule, {
-  valid: [
-    `
-      import { of } from 'rxjs';
-      export class Effect {
-        effect$ = createEffect(
-          () =>
-            this.actions$.pipe(
-              ofType(CollectionApiActions.addBookSuccess),
-              concatLatestFrom(action => this.store.select(fromBooks.getCollectionBookIds)),
-              switchMap(([action, bookCollection]) => {
-                return of({type:'noop'})
-              })
-            ),
-        );
-      }`,
-    `
-      import { of } from 'rxjs';
-      class Effect {
-        detail$ = createEffect(() => {
-          return this.actions.pipe(
-            ofType(ProductDetailPage.loaded),
-            concatMap((action) =>
-              of(action).pipe(withLatestFrom(this.store.select(selectProducts))),
-            ),
-            mergeMap(([action, products]) => {
-            })
-          )
+ruleTester({ ngrxModule: NGRX_MODULE_PATHS.effects, version: '12.1.0' }).run(
+  path.parse(__filename).name,
+  rule,
+  {
+    valid: [
+      {
+        code: `
+import { Actions } from '@ngrx/effects'
```

Comment:
This formatting in tests is something that is widely used in `@typescript-eslint` repo and I think it would be a good idea to use it here as well to avoid trying to format it correctly. It's pretty boring when working with `output` and in general trying to format things manually can even be a real waste of time... so this way we don't need to think about it, we simply align to the left. WDYT?